apps/web/views.py

```diff
@@ -49,12 +47,10 @@ def notify_users_about_challenge(request):
             return render(request, template_name)
 
         elif request.method == 'POST':
-            template_name = 'notification_email.html'
-            emails = User.objects.all().exclude(email__isnull=True, email__exact='').values_list('email', flat=True)
-            htmly = get_template('notification_email.html')
+            users = list(User.objects.exclude(email__exact='').values_list('email', flat=True))
```

Comment:
Do we really need to convert to list? Doesn't `values_list` takes care of that? I am not sure about this though. Please add a comment on the line why we are converting to list if we need to do this.

Reply:
No, for now we don't need that. Earlier `boto` required it to be converted to a list rather than `Queryset` object. :-)